File: runner-client/examples/stop_executor.rs
use tonic::Request;

use runner::runner_client::RunnerClient;
use runner::StopExecutorRequest;

#[tokio::main]
async fn main() -> Result<(), Box<dyn std::error::Error>> {
    let mut client = RunnerClient::connect("http://localhost:50007").await?;

    let response = client
        .stop_executor(Request::new(StopExecutorRequest {
            // Deterministic ID for morgs.near/test
            executor_id: "be21b48c307671c1b3768ed84439f736c1cbbd77f815986354e855d44efd16e6"
                .to_string(),
        }))
        .await?;

    println!("{:#?}", response.into_inner());

    Ok(())
}

File: runner/examples/stop-executor.ts
// Run with 'npx ts-node src/test-client.ts'

import runnerClient from '../src/server/services/runner/runner-client';

runnerClient.StopExecutor({
  executorId: '0293a6b1dcd2259a8be6b59a8cd3e7b4285e540a64a7cbe99639947f7b7e2f9a'
}, (err, response) => {
  if (err) {
    console.error('error: ', err);
  } else {
    console.log('stop request: ', JSON.stringify({ response }, null, 2));
  }
});
